Assets/AnkrSDK/Runtime/MirageAPI/SmartContractManager/Implementation/SmartContractRequests.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using AnkrSDK.MirageAPI.SmartContractManager.Data.AllContracts;
+using AnkrSDK.MirageAPI.SmartContractManager.Data.GetContract;
+using AnkrSDK.MirageAPI.SmartContractManager.Helpers;
+using AnkrSDK.MirageAPI.SmartContractManager.Infrastructure;
+using AnkrSDK.Utils;
+using Cysharp.Threading.Tasks;
+using UnityEngine;
+
+namespace AnkrSDK.MirageAPI.SmartContractManager.Implementation
+{
+	public class SmartContractRequests : ISmartContractRequests
+	{
+		private string _applicationToken;
+
+		public bool IsInitialized()
+		{
+			return !string.IsNullOrEmpty(_applicationToken);
```

Comment:
Not important, but IsNullOrWhitespace actually checks for all three states - null, empty and consisting of whitespaces, so if some user input field was saved with whitespaces then it is safer to use IsNullOrWhitespace